pages/terms.vue

```diff
@@ -2,113 +2,113 @@
     <div class="flex flex-col overflow-y-auto w-full px-10 md:px-32">
         <div>
             <h1
-                class="mb-20 text-black text-5xl font-bold font-['Noto Sans JP']"
+                class="mb-20 text-primary-text text-5xl font-bold font-sans"
             >
                 Terms and Conditions
             </h1>
             <p
-                class="mb-10 text-rose-400 text-3xl font-bold font-['Noto Sans JP']"
+                class="mb-10 text-rose-400 text-3xl font-bold font-sans"
```

Comment:
You should update these colors as well. I think this could be either the muted color or you could do an opacity change like `text-primary/40`

Reply:
The muted is a bit to light compared to the design (on how is is now), I implemented `text-primary-text/55` and it's still not gray but I think using black makes everything more consistent (we use black as text in other components).

![image](https://github.com/ourjapanlife/findadoc-web/assets/114712265/b55fd59f-8d4f-4265-94ad-1b3d75f6985c)